doc/extensibility/IDebugLaunchProvider.md

```diff
@@ -13,6 +13,98 @@ for that purpose:
 3. Pick the "Custom Debugger Extension" template
 4. Follow the instructions on screen.
 
+## Example
+In this example, a simple debugger is added to the project. It will show up in the drop down with the name "MyDebugger"
+when the appropriate capabilities are matched (in this case `MyUnconfiguredProject.UniqueCapability`). The XAML rule
+is completely optional, but it is a convenient way for allowing the user to optionally configure your debugger before
+launching it. The configuration is done by the user setting properties on the user file, then the debugger reads those
+properties while launching. If your debugger has no configuration necessary, then the XAML rule is not needed.
+
+For another example see [ScriptDebuggerLaunchProvider.cs](..\..\samples\WindowsScript\WindowsScript\WindowsScript.ProjectType\ScriptDebuggerLaunchProvider.cs)
```

Comment:
Not very clear which are the 2 samples...  Perhaps you may want to make the 2 tiles below as subtitle to this one?